src/components/TodoNotification/TodoNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import cn from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  currentError: ErrorType,
+  setCurrentError: Dispatch<SetStateAction<ErrorType>>,
```

Comment:
i wrote the same code, but in case of 
`setCurrentError: () => void`

it works )